docs/ref/contrib/gis/gdal.txt

```diff
@@ -1817,6 +1828,67 @@ Here's how to create a raster and return it as a file in an
     ... })
     >>> HttpResponse(rast.vsi_buffer, 'image/tiff')
 
+Using other Virtual Filesystems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Depending on the local build of GDAL other virtual filesystems might be
+supported and can be used by prepending the provided path with the appropriate
+``/vsi*/`` prefix.
+
+.. warning:
+
+    Rasters with names starting with `/vsi*/` will be treated as rasters from
+    the GDAL virtual filesystems. Django doesn't perform any extra validation.
+
+The following sections provide some examples on how to use ``vsi*``
+filesystems, a full reference of each filesystem, alongside possible GDAL
+configuration options, can be found in `GDAL Virtual Filesystems
+documentation.`__
+
+__ https://gdal.org/user/virtual_file_systems.html
+
+Compressed rasters
+^^^^^^^^^^^^^^^^^^
+
+Instead decompressing the file and instantiating the resulting raster, GDAL can
+directly access compressed files using the ``/vsizip/``, ``/vsitar/`` or
+``/vsigz/`` virtual filesystems::
+
+   >>> from django.contrib.gis.gdal import GDALRaster
+   >>> rst = GDALRaster('/vsizip/path/to/your/file.zip/path/to/raster.tif')
+   >>> rst = GDALRaster('/vsitar/path/to/your/file.tar/path/to/raster.tif')
+   >>> rst = GDALRaster('/vsigzip/path/to/your/file.gz')
+
+Network rasters
+^^^^^^^^^^^^^^^
+
+GDAL can support online resources and storage providers transparently. As
+long as it's built with such capabilities.
+
+To access a public raster file with no authentication, use ``/vsicurl/``::
+
+   # Read a remote raster by using /vsicurl/ virtual filesystem
+   >>> from django.contrib.gis.gdal import GDALRaster
+   >>> rst = GDALRaster("/vsicurl/http://example.com/raster.tif")
+   # The name starts with /vsicurl. This raster was read from network
+   >>> rst.name
+   '/vsicurl/http://example.com/raster.tif
+
+
+For commercial storage providers the system should be previously configured for
+authentication and possibly other settings. Refer to `GDAL Virtual Filesystems
+documentation.`__ for each virtual filesystem specific options. Once
+configured, access to the rasters is the same as any other remote raster::
+
+   # Access to a raster in Amazon's S3 cloud store
+   >>> from django.contrib.gis.gdal import GDALRaster
+   >>> rst = GDALRaster(f"/vsis3/path/to/raster.tif")
+   # The name starts with /vsicurl. This raster was read from network
+   >>> rst.name
+
+
+__ https://gdal.org/user/virtual_file_systems.html
```

Comment:
I think it's too details, folks can always check the GDAL docs for more details. I would limit this to few examples, e.g.
```
Network rasters
^^^^^^^^^^^^^^^

GDAL can support online resources and storage providers transparently. As
long as it's built with such capabilities.

To access a public raster file with no authentication, you can use
``/vsicurl/``::

   >>> from django.contrib.gis.gdal import GDALRaster
   >>> rst = GDALRaster('/vsicurl/http://example.com/raster.tif')
   >>> rst.name
   '/vsicurl/http://example.com/raster.tif'

For commercial storage providers (e.g. ``/vsis3``) the system should be
previously configured for authentication and possibly other settings (see
the `GDAL Virtual Filesystems documentation`_ for available options).

.. _`GDAL Virtual Filesystems documentation`: https://gdal.org/user/virtual_file_systems.html
```